.github/ci-init.sh

```diff
@@ -0,0 +1,8 @@
+echo "Installing Quarkus CLI"
+jbang app install --force --name quarkus "io.quarkus:quarkus-cli:${QUARKUS_VERSION}:runner"
```

Comment:
Just for reference https://github.com/quarkusio/quarkus-ecosystem-ci/blob/main/setup-and-test#L104 passes the property